packages/manager/modules/pci/src/projects/project/instances/instance/shelve/translations/Messages_fr_FR.json

```diff
@@ -0,0 +1,8 @@
+{
+  "pci_projects_project_instances_instance_shelve_title": "Suspension de votre instance",
+  "pci_projects_project_instances_instance_shelve_submit_label": "Confirmer",
+  "pci_projects_project_instances_instance_shelve_cancel_label": "Annuler",
+  "pci_projects_project_instances_instance_shelve_content": "Vous allez suspendre votre instance {{ instance }}. Les ressources dédiées à votre instance Public Cloud seront relâchées. Les données de votre disque local seront stockées, la durée de l'opération est fonction de la taille de mon disque local. Je peux redémarrer mon instance à tout moment. Dans l'intervalle, les instances consommées à l'heure ne seront pas facturées. Le stockage utilisé pour stocker mes données sera facturé.",
```

Comment:
`votre` VS `mon` ?

Reply:
True, but for trad Jacques will check them on QC.
I just copy-past the trad that he provide to me.
Thanks.

Reply:
Resolved.